src/App.js

```diff
@@ -1,23 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import HeaderComponent from "./component/Header-section/header.js";
```

Comment:
When you're importing a js file it can be mentioned without the file extension. 
`import HeaderComponent from "./component/Header-section/header";`

Reply:
I tried that but it says module not find, so i wrote the same code back.